pyproject.toml

```diff
@@ -15,37 +15,30 @@ repository = "https://github.com/Idein/actfw-gstreamer"
 version = "0.1.1"
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
+python = "^3.7"
 
-# See also the comment in https://github.com/Idein/actfw-core/blob/master/pyproject.toml .
-#
-# The table of current versions provided by apt:
-#
-# |                             | Raspberry Pi | Jetson Nano |
-# | --------------------------- | ------------ | ----------- |
-# | pithon3-pil (Pillow)        | 5.4.1        | 5.1.0       |
-# | python3-gst-1.0 (PyGObject) | 3.30.4       | 3.26.1      |
-#
-Pillow = "^5"
```

Comment:
We don't need to restrict Pillow version to apt repository one because Pi Zero (armv6) support is dropped.

c.f. https://github.com/Idein/actfw-core/pull/94